pkgs/by-name/te/tenv/package.nix

```diff
@@ -38,9 +38,9 @@ buildGoModule rec {
 
   meta = {
     changelog = "https://github.com/tofuutils/tenv/releases/tag/v${version}";
-    description = "A version manager for OpenTofu, Terraform and Terragrunt written in Go";
+    description = "OpenTofu, Terraform, Terragrunt and Atmos version manager, written in Go.";
     homepage = "https://github.com/tofuutils/tenv";
```

Comment:
```suggestion
    homepage = "https://tofuutils.github.io/tenv/";
```

Since you have a dedicated homepage now?

Reply:
Yes, good idea to change